beacon_node/network/src/sync/network_context.rs

```diff
@@ -142,13 +153,43 @@ impl<T: BeaconChainTypes> SyncNetworkContext<T> {
             request_id: 1,
             blocks_by_root_requests: <_>::default(),
             blobs_by_root_requests: <_>::default(),
+            data_columns_by_root_requests: <_>::default(),
             range_blocks_and_blobs_requests: FnvHashMap::default(),
             network_beacon_processor,
             chain,
             log,
         }
     }
 
+    // TODO(das): epoch argument left here in case custody rotation is implemented
+    pub fn get_custodial_peers(&self, _epoch: Epoch, column_index: ColumnIndex) -> Vec<PeerId> {
+        let mut peer_ids = vec![];
+
+        for (peer_id, peer_info) in self.network_globals().peers.read().connected_peers() {
+            if let Some(enr) = peer_info.enr() {
+                // TODO(das): ignores decode errors
+                let custody_subnet_count = enr
+                    .custody_subnet_count::<T::EthSpec>()
+                    .unwrap_or(T::EthSpec::min_custody_requirement() as u64);
```

Comment:
The method already fallsback to min custody requirement. An error here means there's an issue decoding ENR, I'm not sure if we should fallback to min custody requirement here? I think we should at least log an error here to avoid a hard to find bug.

I think we could probably fallback to `min_custody_requirment` though if ENR fails to decode, providing that we do the same everywhere else.

Reply:
Had a chat with Lion offline and it makes sense to ignore peer ENR decoding error:

> RE ENR decode, can decoding fail due to the other party creating a bad ENR? In that case we should ignore the value